src/bidiMapper/OutgoindBidiMessage.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2021 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Message} from './domains/protocol/bidiProtocolTypes';
+
+export class OutgoingBidiMessage {
+  readonly #message: Message.OutgoingMessage;
+  readonly #channel: string | null;
+
+  constructor(message: Message.OutgoingMessage, channel: string | null) {
```

Comment:
Note out of scope: the constructor can be private

Reply:
Done